### ISSUE TITLE
BOUT++ installs the broken link /usr/local/bin/bout-squashoutput

### ISSUE THREAD

**Issue by yurivict** [boutproject/BOUT-dev#2623]
```
$ ls -l /usr/local/bin/bout-squashoutput
lrwxr-xr-x  1 root  wheel  66 Dec 23 21:05 /usr/local/bin/bout-squashoutput -> ../externalpackages/boutdata/boutdata/scripts/bout_squashoutput.py
```

Instead users should install ```boutdata```.

**Comment by bendudson** [comment]
Thanks @yurivict for going through all this and posting these issues. It might take a while to get to all of them but this is very helpful: I think we have developers running a variety of Linux distributions and Macs, but it's been a while since I regularly used FreeBSD.

**Comment by yurivict** [comment]
The patches that work around the problems are in the FreeBSD ports:
* https://cgit.freebsd.org/ports/tree/science/bout++
* https://cgit.freebsd.org/ports/tree/science/bout++-merging-filaments

**Comment by dschwoerer** [comment]
Did you test the next branch?

https://github.com/boutproject/BOUT-dev/blame/next/CMakeLists.txt#L777

**Comment by dschwoerer** [comment]
Did you use the 4.4.2 release?
I would suggest trying 5.0.0a0.dev10461 (you can get tarballs [here](https://pypi.org/project/boutpp-nightly/#files))

**Comment by dschwoerer** [comment]
> The patches that work around the problems are in the FreeBSD ports:
> 
>     * https://cgit.freebsd.org/ports/tree/science/bout++
> 
>     * https://cgit.freebsd.org/ports/tree/science/bout++-merging-filaments

It would be great if you could open pull requests with those changes :-)

**Comment by dschwoerer** [comment]
I think the question is, should we try to get this fixed in master/4.4.3 or isbit dine to fix it in next/5.0.0 nly?
What do ypu think @yurivict?

**Comment by dschwoerer** [comment]
v5.0.0 is out, and with that we have a release that has a fix
